packages/content-management/lib/index.legacy.ts

```diff
@@ -0,0 +1,28 @@
+// Polyfills for IE9, IE10 & IE11
+import 'es6-promise/auto';
+import 'core-js/es/symbol';
+import 'core-js/es/object';
+import 'core-js/es/function';
+import 'core-js/es/parse-int';
+import 'core-js/es/parse-float';
+import 'core-js/es/number';
+import 'core-js/es/math';
+import 'core-js/es/string';
+import 'core-js/es/date';
+import 'core-js/es/array';
+import 'core-js/es/array';
```

Comment:
This is imported twice

Reply:
Oh, good point, thanks! I'll probably also test which exact polyfills are needed so that we may remove some.